src/store/apiLinks.js

```diff
@@ -0,0 +1,3 @@
+export const moviesLink = () => {
```

Comment:
you don't really use this file

Premise of it is good, I'd add more links here and made it to object instead of individual functions

Reply:
Premise was to have different api endpoints for each different request. Hopefully I'll use it more in the future